src/elements/tokens/TopMovers.tsx

```diff
@@ -4,9 +4,16 @@ import { Token } from 'services/observables/tokens';
 import { prettifyNumber } from 'utils/helperFunctions';
 import { orderBy } from 'lodash';
 
-export const TopMovers = () => {
+interface Props {
+  setSearch: Function;
+}
+
+export const TopMovers = ({ setSearch }: Props) => {
   const tokens = useAppSelector<Token[]>((state) => state.bancor.tokens);
   const topTokens = orderBy(tokens, 'price_change_24', 'desc').slice(0, 20);
+  const handleClick = (token: Token) => {
+    setSearch(token.name);
```

Comment:
expected: search input should contain the symbol

Reply:
done .

Reply:
done .